### PR TITLE
read and write compressed numpy array

### PR DESCRIPTION
**Pull request by comtois-etienne**

### CODE REVIEW

File: linumpy/io/npz.py
Comment (joe-from-mtl) [inline linumpy/io/npz.py:1]: Can you add typeints and documentation for each function you added? We will soon make the repo public and we'll generate doc on readthedocs.

File: linumpy/io/npz.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+
+def write_numpy(npz_path, *, data=None, metadata=None):
```
Comment (joe-from-mtl) [inline linumpy/io/npz.py]: What's the purpose of the second argument?

File: linumpy/io/npz.py
Comment (comtois-etienne) [reply linumpy/io/npz.py:1]: done

File: linumpy/io/npz.py (hunk 1 above)
Comment (comtois-etienne) [reply linumpy/io/npz.py]: it is used to enforce the use of the variable names

**Pull request merged**